README.md

```diff
@@ -1,53 +1,48 @@
 # 🥧 PIE (PHP Installer for Extensions)
 
-You will need PHP 8.1 or newer to run PIE, but PIE can install an extension to any installed PHP version.
+You will need PHP 8.1 or newer to run PIE, but PIE can install an extension to
+any installed PHP version.
 
-## Installing
+If you are an extension maintainer wanting to add PIE support to your extension,
+please read [extension-maintainers](./docs/extension-maintainers.md).
 
-## Usage
+## Installing an extension using PIE
 
-You can download an extension ready to be built or installed using the `download` command. For example, to download the
-`example_pie_extension` extension, you would run:
+You can install an extension using the `install` command. For example, to
+install the `example_pie_extension` extension, you would run:
 
 ```shell
-$ bin/pie download asgrim/example-pie-extension
-You are running PHP 8.3.7
-Target PHP installation: 8.3.7 nts, on Linux/OSX/etc x86_64 (from /usr/bin/php8.3)
+$ bin/pie install asgrim/example-pie-extension
```

Comment:
Perhaps it'd be wise to use `example/` instead of `asgrim/`?

Reply:
I was copy/pasta from my terminal to ensure we had "real" output, but yeah, I'll replace this :+1: